### PR TITLE
Fix breadcrumbs for various apps

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This ticket fixes the missing breadcrumbs on the Burials Benefits app, but I also found a few other breadcrumb inconsistencies along the way, so I patched those too.

Ticket - https://github.com/department-of-veterans-affairs/va.gov-team/issues/6381

## Testing done
- Looked at the generated breadcrumbs for every affected page

## Screenshots
image

## Acceptance criteria
- [x] Breadcrumbs are rendered correctly

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/burials/manifest.json [hunk 1]
```diff
@@ -8,6 +8,21 @@
...
+      },
+      {
+        "path": "burials-memorials/veterans-burial-allowance",
+        "name": "Burial allowance"
+      },
+      {
+        "path": "burials-and-memorials/application/530",
+        "name": "Apply for Burial Benefits (VA Form 21P-530)"
+      }
+    ]
```
Comment (ncksllvn) [inline src/applications/burials/manifest.json]: image

File: src/applications/edu-benefits/0993/manifest.json [hunk 2]
```diff
@@ -15,6 +15,10 @@
         "path": "education/",
         "name": "Education and training"
       },
+      {
+        "path": "education/opt-out-information-sharing/",
+        "name": "Opt out of information sharing"
+      },
```
Comment (ncksllvn) [inline src/applications/edu-benefits/0993/manifest.json]: image

File: src/applications/edu-benefits/1990/manifest.json [hunk 3]
```diff
@@ -16,8 +16,12 @@
         "path": "education/"
       },
       {
-        "name": "Apply for education benefits",
+        "name": "How to apply",
         "path": "education/how-to-apply/"
+      },
+      {
+        "name": "Apply for education benefits",
+        "path": "education/apply-for-education-benefits/application/1990"
```
Comment (ncksllvn) [inline src/applications/edu-benefits/1990/manifest.json]: image

File: src/applications/edu-benefits/1990e/manifest.json [hunk 4]
```diff
@@ -19,8 +19,12 @@
         "path": "education/"
       },
       {
-        "name": "Apply for education benefits",
+        "name": "How to apply",
         "path": "education/how-to-apply/"
+      },
+      {
+        "name": "Apply for education benefits",
+        "path": "education/apply-for-education-benefits/application/1990E"
```
Comment (ncksllvn) [inline src/applications/edu-benefits/1990e/manifest.json]: image

File: src/applications/edu-benefits/1990n/manifest.json [hunk 5]
```diff
@@ -16,8 +16,12 @@
         "path": "education/"
       },
       {
-        "name": "Apply for education benefits",
+        "name": "How to apply",
         "path": "education/how-to-apply/"
+      },
+      {
+        "name": "Apply for education benefits",
+        "path": "education/apply-for-education-benefits/application/1990N"
```
Comment (ncksllvn) [inline src/applications/edu-benefits/1990n/manifest.json]: image

File: src/applications/edu-benefits/1995/manifest.json [hunk 6]
```diff
@@ -18,8 +18,12 @@
         "path": "education/"
       },
       {
-        "name": "Apply for education benefits",
+        "name": "How to apply",
         "path": "education/how-to-apply/"
+      },
+      {
+        "name": "Apply for education benefits",
+        "path": "education/apply-for-education-benefits/application/1995"
```
Comment (ncksllvn) [inline src/applications/edu-benefits/1995/manifest.json]: image

File: src/applications/edu-benefits/5490/manifest.json [hunk 7]
```diff
@@ -19,8 +19,12 @@
         "path": "education/"
       },
       {
-        "name": "Apply for education benefits",
+        "name": "How to apply",
         "path": "education/how-to-apply/"
+      },
+      {
+        "name": "Apply for education benefits",
+        "path": "education/apply-for-education-benefits/application/5490"
```
Comment (ncksllvn) [inline src/applications/edu-benefits/5490/manifest.json]: image

File: src/applications/hca/manifest.json [hunk 8]
```diff
@@ -23,6 +23,10 @@
       },
       {
         "path": "health-care/how-to-apply/",
+        "name": "How to apply"
+      },
+      {
+        "path": "health-care/apply/application",
```
Comment (ncksllvn) [inline src/applications/hca/manifest.json]: image

File: src/applications/pensions/manifest.json [hunk 9]
```diff
@@ -21,6 +21,10 @@
       },
       {
         "path": "pension/how-to-apply/",
+        "name": "How to apply"
+      },
+      {
+        "path": "pension/application/527EZ",
```
Comment (ncksllvn) [inline src/applications/pensions/manifest.json]: image

File: src/applications/veteran-id-card/manifest.json [hunk 10]
```diff
@@ -11,6 +11,14 @@
...
     "includeBreadcrumbs": true,
     "breadcrumbs_override": [
+      {
+        "path": "records/",
+        "name": "Records"
+      },
+      {
+        "path": "records/get-veteran-id-cards/",
+        "name": "Types of Veteran ID cards"
+      },
```
Comment (ncksllvn) [inline src/applications/veteran-id-card/manifest.json]: image

File: src/applications/edu-benefits/0994/manifest.json [hunk 11]
```diff
@@ -10,12 +10,24 @@
...
+        "name": "How to use your benefits",
+        "path": "education/about-gi-bill-benefits/how-to-use-benefits/"
+      },
+      {
+        "name": "VET TEC program",
+        "path": "education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/"
+      },
+      {
+        "name": "Apply for education benefits",
+        "path": "education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/apply-for-vettec-form-22-0994"
```
Comment (ncksllvn) [inline src/applications/edu-benefits/0994/manifest.json]: image

**Review by kelsonic — APPROVED** [review]
Thanks for leaving screenshots by each! Nicely done!! 🎉

**Review by andaleliz — COMMENTED** [review]
Wow, thanks for jumping on this! I'm not totally clear on what the final output will be because the screen shows the breadcrumb reading "Apply now" but I see `Apply for Burial Benefits (VA Form 21P-530)` in the JSON.

I believe the breadcrumb should read `Apply for burial benefits`, following the sentence-case pattern we see on the education page example.

**Comment by ncksllvn** [comment]
Thanks @andaleliz! Fixed. This is why I asked for your review 😄

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**